odl/contrib/datasets/ct/mayo.py

```diff
@@ -252,13 +255,15 @@ def load_reconstruction(folder, slice_start=0, slice_end=-1):
         volumes.append(densities)
         datasets.append(dataset)
 
-    # Compute geometry parameters
     voxel_size = np.array(list(pixel_size) + [pixel_thickness])
     shape = np.array([rows, cols, len(volumes)])
-    min_pt = (np.array(dataset.ImagePositionPatient) -
-              np.array(dataset.DataCollectionCenterPatient))
 
-    max_pt = min_pt + voxel_size * np.array([rows, cols, 0])
+    # Compute geometry parameters
+    mid_pt = (np.array(dataset.ReconstructionTargetCenterPatient) -
+              np.array(dataset.DataCollectionCenterPatient))
```

Comment:
Seems like DICOM has a lot of (potentially conflicting) redundant settings.

Reply:
It's a nightmare